### PR TITLE
Previous button get the same preview window as the next song button

### PR DESCRIPTION
**Pull request by vsarunov**
Added a tooltip to show the previous song information if it exists. Weird spacing though, does only appear when I compare files.

### CODE REVIEW

**Comment by Bond-009** [comment]
@vsarunov The spacing is because you use tabs and not spaces

**Comment by vsarunov** [comment]
Thanks for pointing out, fixed it manually.

File: BreadPlayer.Views.UWP/ViewModels/ShellViewModel.cs [hunk 1]
```diff
@@ -508,6 +512,13 @@ public Mediafile UpcomingSong
             get { return upcomingsong; }
             set { Set(ref upcomingsong, value); }
         }
+
+        Mediafile previoussong = new Mediafile(); //we init beforehand so no null exception occurs
+        public Mediafile PreviousSong
+				{
```
Comment (Bond-009) [inline BreadPlayer.Views.UWP/ViewModels/ShellViewModel.cs]: Still some tabs

**Comment by theweavrs** [comment]
Okay, thanks for the PR. I will merge this and try it out! Thanks again.

**Comment by vsarunov** [comment]
@theweavrs I am looking at some other issues that I found not to be solved in the list.

**Comment by theweavrs** [comment]
@vsarunov Please do, would really appreciate the help. I already have my hands full with Beta feature-list. If you need any help, I would be glad to lend you a hand :)

**Pull request merged**